Modules/Twp/Database/Migrations/2024_02_08_232612_add_student_indigeneity_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::connection(env('DB_DATABASE_TWP'))->create('indigeneity_student', function (Blueprint $table) {
+            $table->foreignId('student_id')->constrained('students')->onDelete('cascade');
+            $table->foreignId('indigeneity_id')->constrained('indigeneity_types')->onDelete('cascade');
```

Comment:
rename indigeneity_id to indigeneity_type_id

Reply:
rename the file to ... create_indigeneity_type_student_table
rename the table to indigeneity_type_student and it's reference in the models